pkg/virt-controller/watch/workload-updater/workload-updater_test.go

```diff
@@ -444,6 +429,12 @@ var _ = Describe("Workload Updater", func() {
 	})
 })
 
+func waitForVMIInformer(wu *WorkloadUpdateController, vmisNo int) {
+	EventuallyWithOffset(1, func() []interface{} {
+		return wu.vmiInformer.GetStore().List()
+	}, 3*time.Second, 200*time.Millisecond).Should(HaveLen(vmisNo))
```

Comment:
I think this func should be renamed: when I saw the function calls, I didn't immediately realize the 2nd parameter was the number of expected VMIs in the informer cache - I somehow assumed it would be something related to time.

All I can come up with is `waitForNumberOfInstancesOnVMIInformerCache`. loooooooong name, which I am not so fond of. But I think it would address my personal issue with the function.

Reply:
Yes, definitely not the best name, I've adopted your solution since I really can't think of anything better, thank you!